Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,10 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+
+function toUpperSnakeCase(string) {
+    return string.split(" ").join("_").toUpperCase();
+}
+
+console.log(toUpperSnakeCase("Hello World"));
```

Comment:
What assertion could you write against this function to prove it works?

Reply:
I wrote some assertions.